src/dev/ci_setup/setup.sh

```diff
@@ -104,6 +104,14 @@ yarnGlobalDir="$(yarn global bin)"
 export PATH="$PATH:$yarnGlobalDir"
 hash -r
 
+###
+### use the chromedriver cache if it exists
+###
+if [ -f "$dir/.chromedriver/2.42/chromedriver_linux64.zip" ]; then
```

Comment:
I think we can automatically get the chromedriver version we need to download, with something like:

`$($nodeBin/node -p -e "require('./package.json').devDependencies.chromedriver.split('.').slice(0,2).join('.'))`

What do u think @spalger ?

Reply:
I'm not 100% sure if the version from the package.json will match the version here, all I know is that it's currently downloading 2.42... Maybe we should use `master` for the version directory and just point it to whatever version the chromedriver package downloads to `node_modules/chromedriver/bin/chromedriver`. I'm going to give that a shot so we're not bound to a specific version.

Reply:
yeah makes sense! 👍